keyboards/kudox_full/post_config.h

```diff
@@ -0,0 +1,7 @@
+#ifndef MOUSEKEY_ENABLE
+    #define MOUSEKEY_INTERVAL       20
+    #define MOUSEKEY_DELAY          0
+    #define MOUSEKEY_TIME_TO_MAX    60
+    #define MOUSEKEY_MAX_SPEED      7
+    #define MOUSEKEY_WHEEL_DELAY    0
+#endif
```

Comment:
```suggestion
// Copyright 2021 Kumao Kobo (@kumaokobo)
// SPDX-License-Identifier: GPL-2.0+

#pragma once

#ifndef MOUSEKEY_INTERVAL
#    define MOUSEKEY_INTERVAL       20
#endif
#ifndef MOUSEKEY_DELAY
#    define MOUSEKEY_DELAY          0
#endif
#ifndef MOUSEKEY_TIME_TO_MAX
#    define MOUSEKEY_TIME_TO_MAX    60
#endif
#ifndef MOUSEKEY_MAX_SPEED
#    define MOUSEKEY_MAX_SPEED      7
#endif
#ifndef MOUSEKEY_WHEEL_DELAY
#    define MOUSEKEY_WHEEL_DELAY    0
#endif
```

This is what I meant, specifically.  This should allow users to set different values, but default to these, if none are set.

Reply:
Thank you. That makes sense!